frontend/src/metabase/public/containers/PublicDashboard.jsx

```diff
@@ -157,6 +158,9 @@ class PublicDashboard extends Component {
       ? getDashboardActions(this, { ...this.props, isPublic: true })
       : [];
 
+    const isNotAction = dashcard => !dashcard.action;
+    const visibleDashcards = (dashboard?.dashcards ?? []).filter(isNotAction);
+
```

Comment:
Could we please use the [`isActionDashCard` type guard](https://github.com/metabase/metabase/blob/83d79b24cfdb97f42f08302f4b5bca818ae911c4/frontend/src/metabase/dashboard/utils.ts#L97) here?

Reply:
Awesome. This is very handy. Thanks ❤️